serine.galstyan/repo/Lesson_3/exercise_13/exercise_13.py

```diff
@@ -0,0 +1,8 @@
+#!/usr/bin/evn python3
+
+meal_cost = int(input("Enter the cost of the meal:"))
+tax_amount = meal_cost * 9 / 100
+tip_amount = meal_cost * 18 / 100
+grand_total = meal_cost + tax_amount + tip_amount
+print("The tax amount is:", round(tax_amount, 2), "\nThe tip amount is:", round(tip_amount, 2), 
```

Comment:
LT: Rounding is a bad variant to represent the decimal point. It rounds the real value after the point (fractional part). For example round(49.6978, 2) will return 49.7 when the real two decimal point representation should return 49.69.

Hope this will be helpful. Next time do not use the round() function without good reason.